tools/mmp/driver.cs

```diff
@@ -408,9 +408,20 @@ static void Main2 (string [] args)
 			if (!targetFramework.HasValue)
 				targetFramework = TargetFramework.Default;
 
-			if (TargetFramework.Identifier == TargetFramework.Xamarin_Mac_2_0.Identifier) {
+			if (TargetFramework == TargetFramework.Xamarin_Mac_2_0_Mobile || TargetFramework.Version == TargetFramework.Xamarin_Mac_2_0_Mobile.Version) {
```

Comment:
Is it in the tests that we're manually creating that framework or in our code somewhere?  And is that something we could change to always create the target framework?